### ISSUE TITLE
Generic command is not functional to run gateway commands via NMS.

### ISSUE THREAD

**Issue by khansiddiquekc** [magma/magma#13894]
### Your Environment

- **Version:** 1.7.0-1649243834-f237923d
- **Affected Component:** Orchestrator, Access Gateway, or NMS
- **Deployment Environment:** Bare-metal (AGW).

### Describe the Issue

The Command option in the NMS (Equipment -> Gateway -> specific_gateway -> Actions->Command) includes "Ping" and "Generic", where "Ping" is working fine and able to ping AGW, but the "Generic" command is failing to execute.

#### To Reproduce

Step 1: In NMS, go to 'Equipment -> Gateway -> specific_gateway -> Actions->Command'
Step 2: Give any "command name" in the Generic option.
The command is failing to execute.

### Expected behavior

The Generic command should be operational and executable gateway commands. 

#### Screenshots

![image.png](https://images.zenhubusercontent.com/619377abf0934478f7dd8a48/1f39d49b-8059-46ef-afe6-2d97f59c8a19)
Able to ping the Gateway:
![image.png](https://images.zenhubusercontent.com/619377abf0934478f7dd8a48/f41950b7-d33c-4c4d-9929-d9bc915cba46)
Failing to execute the command:
![image.png](https://images.zenhubusercontent.com/619377abf0934478f7dd8a48/9f333a74-37a8-49b7-95c2-ad4998ebebc9)
![image.png](https://images.zenhubusercontent.com/619377abf0934478f7dd8a48/1279a0ad-5206-4e11-8008-f15e89b598c5)

**Comment by sebathomas** [comment]
I just reproduced the bug on the current master. Looks like it might be a magmad issue, I see in the logs that the command is forwarded to magmad's `rpc_servicer.py` before it fails.

**Comment by sebathomas** [comment]
I think this might be a documentation problem and the error handling here gives no feedback to the user. It seems that the list of allowed generic commands has to be given in the magmad config (`generic_command_config`), and other commands will fail with a 500.

Did you configure any allowed commands for magmad?

**Comment by sebathomas** [comment]
Tasks:
- Add better error handling to magmad and orc8r and show a sensible error message in the frontend.
- Add documentation in the frontend, e.g. a short description in the command pop-up window.

**Comment by voisey** [comment]
@khansiddiquekc Some improvements to this functionality have been merged to master via https://github.com/magma/magma/pull/14014, which should hopefully make it less confusing in the future. As @sebathomas alluded to, the allowed commands are configured in `magmad.yml`, hence the commands you tried not working. I've also opened an [issue](https://github.com/magma/magma/issues/14040) to do a more general refactoring of this functionality.
